README.md

```diff
@@ -73,15 +73,9 @@ Please refer to [our website](https://www.precice.org/adapter-fenics.html#how-ca
 
 To create and install the `fenicsprecice` python package the following instructions were used: [How To Package Your Python Code from python-packaging.readthedocs.io](https://python-packaging.readthedocs.io/en/latest/index.html).
 
-## Citing
+## References & Citing
```

Comment:
"References" has been added, but there is no reference to "references" in the text. I will make a suggestion for the text below.

Maybe we want to chose a slightly different name for this section?
```suggestion
## How to cite and further literature
```